notebooks/rag_eval_ragas.ipynb

```diff
@@ -0,0 +1,5272 @@
+{
```

Comment:
<p>Why not do all at once - for example all metrics that don't require ground truth or need common fields?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
<p>I'm not entirely clear on your question. Could you provide more details?</p><p>I included this section to demonstrate the computation of multiple metrics using a single pipeline object.</p>